src/main/java/lotto/domain/Ranking.java

```diff
@@ -0,0 +1,51 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum Ranking {
+
+    FIRST(2000000000, 6, 0),
+    SECOND(30000000, 5, 1),
+    THIRD(1500000, 5, 0),
+    FOURTH(50000, 4, 0),
+    FIFTH(5000, 3, 0),
+    FAIL(0, 0, 0);
```

Comment:
아래와 같이 3자리 숫자마다 `_`를 붙여서 가독성을 높일 수 있어요! 🤗 
```suggestion
    FIRST(2_000_000_000, 6, 0),
    SECOND(30_000_000, 5, 1),
    THIRD(1_500_000, 5, 0),
    FOURTH(50_000, 4, 0),
    FIFTH(5_000, 3, 0),
    FAIL(0, 0, 0);
```

Reply:
오왓! 꿀팁 감삼다 ㅎㅎ 🙇🏻